chore: update experimental stability definition, formatting fixes (#283)
snyk/sweater-comb: docs/principles/version.md

In increasing order of stability, from least to most stable, these are:

- `wip`

>     - The resource is just getting started and functionally may be in a broken or incomplete state with regard to its OpenAPI spec. WIP-versioned resources must not be made publicly accessible.

Reviewer: ```suggestion
    - The resource is just getting started and may be in a broken or incomplete state with regard to its OpenAPI spec. WIP-versioned resources are not made publicly accessible.
```

- `experimental`
    - The resource version is a functionally complete implementation of its OpenAPI spec. Experimental versions may be publicly accessible.

>     - Comes with NO guarantees on the stability or future availability of the API version.

Reviewer: ```suggestion
    - There are NO guarantees on the stability or future availability of the API version.
```

      - Breaking changes in the API may be introduced at any time.
      - The API version may sunset at any time.
    - May be distributed as a limited tech preview.
- `beta`

>     - The resource version is being evaluated as a serious candidate for general availability by partners and customers.

Reviewer: ```suggestion
    - The resource version is being evaluated as a candidate for general availability by partners and customers.
```

    - During its availability, it must not be revised with incompatible and breaking changes.
    - It may be revised with backwards-compatible, additive changes.

> `experimental` is where new API features are incubated. Typically experiments are unlisted in official product documentation, though they may be publicly routed. They're a place to try new things, stress test, invite early feedback, and iterate. Experiments are a temporary draft of an API, so they are considered deprecated upon release, and are considered sunset, expiring 90 days after their release. By this time, an experiment should iterate with a new release version, or promote to `beta`.

Reviewer: ```suggestion
`experimental` is where new API features are incubated. Experiments are unlisted in official product documentation, though they may be publicly routed. They're a place to try new things, stress test, invite early feedback, and iterate. Experiments are a temporary draft of an API, so they are considered deprecated upon release, and are considered sunset, expiring 90 days after their release. By this time, an experiment should iterate with a new release version, or promote to `beta`.
```
```suggestion
`experimental` is where new API features are incubated. Typically experiments are unlisted in official product documentation, though they may be publicly routed. They're a place to try new things, stress test, invite early feedback, and iterate. Experiments are a temporary draft of an API, so they are considered deprecated upon release, and are considered sunset, expiring 90 days after their release. By this time, an experiment should iterate with a new release version, or promote to `beta`.
```

> `experimental` is where new API features are incubated. Typically experiments are unlisted in official product documentation, though they may be publicly routed. They're a place to try new things, stress test, invite early feedback, and iterate. Experiments are a temporary draft of an API, so they are considered deprecated upon release, and are considered sunset, expiring 90 days after their release. By this time, an experiment should iterate with a new release version, or promote to `beta`.

Reviewer: Not sure what's being said about sunset; that they're already deprecated, so in a sunset state, and will be completely expired in 90? Or that they're deprecated at release, and will be sunset/expired in 90 days? Do we have a clear distinction between expired and sunset?

Author: I'll whittle that down and just say "sunset after 90 days".
